test/requests.js

```diff
@@ -505,22 +505,6 @@ const tests = function(web3) {
     });
   });
 
-  describe("eth_getTransactionReceipt", function() {
```

Comment:
I think maybe this was removed by accident?  This looks like it was part of @eshaben's [commit](https://github.com/trufflesuite/ganache-core/commit/4d3c606852ce7445a35e9767d770386ae5b8f6a8) for PR [212](https://github.com/trufflesuite/ganache-core/pull/212)

Reply:
This may be some squashing magic gone wrong

Reply:
This was definitely from me